### ISSUE TITLE
Handin files not listed on website

### ISSUE THREAD

**Issue by chriswailes** [autolab/Autolab#887]
Using tag 2.0.8 of AutoLab, handin files for the Lab Template are not being displayed on the web page after submission.  In this case, I changed the name of the handin for the assignment to Foo.java.  After then submitting a Foo.java file, the `admin@foo.bar_1_Foo.java` file appeared as expected in the appropriate courses directory.  However, each displayed submission contains an empty file list.

**Comment by chriswailes** [comment]
Digging into the database, it appears that the file name is not getting inserted into the `submissions` table.  Here is the line that was added:

```
1829|1|1|19||2017-08-15 19:16:36.885168|2017-08-15 19:16:36.885168|||0|1|||172.17.0.1||f|||
```

**Comment by chriswailes** [comment]
I've traced the failure to `Submission.save_additional_form_fields`.  When this method is called from `Submission.save_file` the error `bin/rails: No such file or directory - file` is printed to the console, rolls back the transaction, and  throws an error taking control flow out of `save_file`.  The end result is that the submission file name isn't saved to the database.

This leads me to believe that this might be a setup problem on my end, but I can't figure out why.  I've installed AutoLab into `/opt/autolab`, and there is a `bin` directory which contains a `rails` executable.  The startup script for AutoLab is as follows:

```Bash
cd /opt/autolab
bundle exec rails s -b 0.0.0.0 -p 3000
```
